virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -24,48 +32,32 @@ The network attachment definition must be in the same namespace as the pod or vi
 
 [NOTE]
 ====
-There are multiple methods for configuring a VLAN, including network attachment definition and node network configuration policy.
-However, a network attachment definition provides a more efficient and more manageable configuration.
+There are multiple methods for configuring a VLAN, for example, creating a node network configuration policy. However, a network attachment definition provides a more efficient and manageable configuration
 ====
 
-include::modules/virt-networking-glossary.adoc[leveloffset=+1]
-
-[id="virt-creating-linux-bridge"]
-== Configuring a Linux bridge
-
-include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+2]
+include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+1]
 
 For more information about scheduling, interface types, and other node networking activities,
 see the xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking]
 section.
 
 [id="virt-creating-network-attachment-definition"]
-== Creating a network attachment definition
+= Creating a Linux bridge network attachment definition
 
 [WARNING]
 ====
-Configuring ipam in a network attachment definition for virtual machines is not supported.
+Configuring IPAM in a network attachment definition for virtual machines is not supported.
 ====
 
 include::modules/virt-creating-bridge-nad-web.adoc[leveloffset=+2]
```

Comment:
I don't understand why this was created as a virt module. It should be part of networking.

Reply:
We can still do this when we meet on May 11 during your open hours

I think after mod docs were released, logic was sidelined in a lot of cases in that people thought "everything must be module" when as you say, it really depends on the context of the task in question and what the user needs to perform it most effectively. So many of these modules will NEVER be reused.